.github/workflows/release-notes.yaml

```diff
@@ -0,0 +1,53 @@
+# TODO: Add published to CDN date to description when the release is marked prerelease: false
+name: Create GitHub Release
+on:
+  repository_dispatch:
+    types:
```

Comment:
Dope

Thoughts on adding a make command for this like we did with react-recurly?

Reply:
I think it has to wait on the job being able to accept a version parameter from the webhook. The dispatch capability isn't able to specify a tag for the job to consume. So for now, this is just a stand-in.